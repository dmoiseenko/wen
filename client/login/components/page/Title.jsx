import React from 'react';


function Title() {
  return (
    <h1 className="title is-1">wen</h1>
  );
}

export default Title;
