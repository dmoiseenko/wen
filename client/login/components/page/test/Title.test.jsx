import React from 'react';
import renderer from 'react-test-renderer';

import Title from '../Title';


describe('<Title />', () => {
  it('should render', () => {
    const tree = renderer.create(
      <Title />
    );

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
