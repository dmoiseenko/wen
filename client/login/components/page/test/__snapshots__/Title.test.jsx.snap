// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should render 1`] = `
<h1
  className="title is-1"
>
  wen
</h1>
`;
